spec/index.bs

```diff
@@ -556,7 +556,30 @@ dictionary GPULimits {
 A {{GPUBuffer}} represents a block of memory that can be used in GPU operations.
 Data is stored in linear layout, meaning that each byte of the allocation can be
 addressed by its offset from the start of the {{GPUBuffer}}, subject to alignment
-restrictions depending on the operation.
+restrictions depending on the operation. Some {{GPUBuffer}} can be mapped which
+makes the block of memory accessible via an {{ArrayBuffer}} called it's mapping.
```

Comment:
we don't have `{{ArrayBuffer}}` defined, is it automatically hooked up somehow?

Reply:
Yeah there's some bikeshed magic going on. Not sure where though ^^